include/TPP/Dialect/Tpp/TppOps.td

```diff
@@ -107,15 +107,15 @@ def Tpp_ReluOp : Tpp_Op<"relu"> {
 
     ```mlir
  
-    tpp.relu outs(%1: memref<2x2xf32>)
+    tpp.relu out(%1: memref<2x2xf32>)
```

Comment:
most ops in MLIR dialects use `outs`, even when there's a single one. Why change here?

Reply:
This was to be consistent with the other operations that use `out` and not `outs`. No string opinion but we need to be consistent with the other operations.

Reply:
I don't mind either way